Compatibility with Julia 1.1

Project.toml seems to indicate Julia 1.1 is supported, however I'm getting:

```
2020-05-26T00:05:54.0518174Z Switched to branch 'compathelper/new_version/2020-05-26-00-05-54-038-698849552'
2020-05-26T00:05:55.0433918Z ERROR: MethodError: no method matching hasmethod(::typeof(CompatHelper.update_manifests), ::Type{Tuple{}}, ::Tuple{Symbol})
2020-05-26T00:05:55.0434242Z Closest candidates are:
2020-05-26T00:05:55.0508947Z   hasmethod(::Any, ::Any; world) at reflection.jl:1100
2020-05-26T00:05:55.0790278Z Stacktrace:
2020-05-26T00:05:55.3953965Z  [1] #make_pr_for_new_version#14(::GitHub.OAuth2, ::Base.EnvDict, ::Symbol, ::Bool, ::CompatHelper.DefaultBranch, ::String, ::String, ::Array{Pkg.Types.RegistrySpec,1}, ::typeof(CompatHelper.make_pr_for_new_version), ::typeof(CompatHelper.update_manifests), ::String, ::String, ::GitHub.Repo, ::CompatHelper.Package, ::Dict{CompatHelper.Package,Union{Nothing, VersionSpec}}, ::Dict{CompatHelper.Package,Union{Nothing, String}}, ::Dict{CompatHelper.Package,Union{Nothing, VersionNumber}}, ::Array{GitHub.PullRequest,1}, ::Array{String,1}, ::CompatHelper.GitHubActions) at /home/runner/.julia/packages/CompatHelper/eVLge/src/new_versions.jl:306
2020-05-26T00:05:55.4006671Z  [2] #make_pr_for_new_version at ./strings/string.jl:0 [inlined]
2020-05-26T00:05:55.4020325Z  [3] #make_pr_for_new_version#13(::GitHub.OAuth2, ::Base.EnvDict, ::Bool, ::Bool, ::CompatHelper.DefaultBranch, ::String, ::String, ::Array{Pkg.Types.RegistrySpec,1}, ::typeof(CompatHelper.make_pr_for_new_version), ::typeof(CompatHelper.update_manifests), ::GitHub.Repo, ::CompatHelper.Package, ::Dict{CompatHelper.Package,Union{Nothing, VersionSpec}}, ::Dict{CompatHelper.Package,Union{Nothing, String}}, ::Dict{CompatHelper.Package,Union{Nothing, VersionNumber}}, ::Array{GitHub.PullRequest,1}, ::Array{String,1}, ::CompatHelper.GitHubActions) at /home/runner/.julia/packages/CompatHelper/eVLge/src/new_versions.jl:135
2020-05-26T00:05:55.4020587Z  [4] #make_pr_for_new_version at ./array.jl:0 [inlined]
2020-05-26T00:05:55.4020846Z  [5] #make_pr_for_new_version#12(::GitHub.OAuth2, ::Base.EnvDict, ::Bool, ::Bool, ::CompatHelper.DefaultBranch, ::String, ::String, ::Array{Pkg.Types.RegistrySpec,1}, ::typeof(CompatHelper.make_pr_for_new_version), ::typeof(CompatHelper.update_manifests), ::GitHub.Repo, ::Dict{CompatHelper.Package,Union{Nothing, VersionSpec}}, ::Dict{CompatHelper.Package,Union{Nothing, String}}, ::Dict{CompatHelper.Package,Union{Nothing, VersionNumber}}, ::Set{CompatHelper.Package}, ::Array{GitHub.PullRequest,1}, ::Array{String,1}, ::CompatHelper.GitHubActions) at /home/runner/.julia/packages/CompatHelper/eVLge/src/new_versions.jl:24
2020-05-26T00:05:55.5046358Z  [6] (::getfield(CompatHelper, Symbol("#kw##make_pr_for_new_version")))(::NamedTuple{(:auth, :env, :keep_existing_compat, :drop_existing_compat, :master_branch, :subdir, :pr_title_prefix, :registries),Tuple{GitHub.OAuth2,Base.EnvDict,Bool,Bool,CompatHelper.DefaultBranch,String,String,Array{Pkg.Types.RegistrySpec,1}}}, ::typeof(CompatHelper.make_pr_for_new_version), ::Function, ::GitHub.Repo, ::Dict{CompatHelper.Package,Union{Nothing, VersionSpec}}, ::Dict{CompatHelper.Package,Union{Nothing, String}}, ::Dict{CompatHelper.Package,Union{Nothing, VersionNumber}}, ::Set{CompatHelper.Package}, ::Array{GitHub.PullRequest,1}, ::Array{String,1}, ::CompatHelper.GitHubActions) at ./none:0
2020-05-26T00:05:55.5046803Z  [7] #main#1(::Array{Pkg.Types.RegistrySpec,1}, ::Bool, ::Bool, ::CompatHelper.DefaultBranch, ::String, ::Array{String,1}, ::Function, ::Function, ::Base.EnvDict, ::CompatHelper.GitHubActions) at /home/runner/.julia/packages/CompatHelper/eVLge/src/main.jl:48
2020-05-26T00:05:55.5281066Z  [8] main at /home/runner/.julia/packages/CompatHelper/eVLge/src/main.jl:16 [inlined] (repeats 4 times)
2020-05-26T00:05:55.5281885Z  [9] top-level scope at none:0
2020-05-26T00:05:55.6315765Z ##[error]Process completed with exit code 1.
```

using Julia 1.1.1

--- a/Project.toml
+++ b/Project.toml
@@ -1,7 +1,7 @@
 name = "CompatHelper"
 uuid = "aa819f21-2bde-4658-8897-bab36330d9b7"
 authors = ["Dilum Aluthge <dilum@aluthge.com>", "Brown Center for Biomedical Informatics"]
-version = "1.8.2"
+version = "1.9.0"
 
 [deps]
 Base64 = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
@@ -17,7 +17,7 @@ UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 GitHub = "5.1.3"
 JSON = "0.21"
 TimeZones = "0.10, 0.11, 1.0"
-julia = "1.1"
+julia = "1.2"
 
 [extras]
 Base64 = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"

